xbstrap/vcs_utils.py

```diff
@@ -17,6 +18,16 @@ class RepoStatus(Enum):
     OUTDATED = 2
 
 
+CHECKSUM_BLOCK_SIZE = 1048576
+HASHLIB_MAP = {
+    "md5": hashlib.md5,
+    "sha1": hashlib.sha1,
```

Comment:
fwiw I'd also drop these two since they aren't good enough for authentication anymore and also aren't necessary to support, and then rebase+merge